.travis.yml

```diff
@@ -0,0 +1,35 @@
+language: java
+sudo: false
+
+os: linux
+dist: bionic
+jdk: openjdk11
+
+cache:
+  directories:
+    - $HOME/.m2
+
+services:
+  - xvfb
+
+addons:
+  apt:
+    packages:
+      - at-spi2-core
+      - build-essential
+      - libgtk-3-dev
+      - libxtst-dev
+
+before_script:
+  - mkdir -p $HOME/graalvm && cd $HOME/graalvm
+  - wget https://github.com/graalvm/graalvm-ce-builds/releases/download/vm-19.3.0/graalvm-ce-java11-linux-amd64-19.3.0.tar.gz
```

Comment:
this is ok for linux, but won't work for Android (but we'll keep it, as I hope the 20 release should work for android)

Reply:
We can always update this once we start targeting Android.